torch/csrc/jit/script/builtin_functions.h

```diff
@@ -7,8 +7,8 @@ namespace torch {
 namespace jit {
 namespace script {
 
-TORCH_API const std::vector<Function*>& getAllBuiltinFunctionsFor(Symbol name);
-
+TORCH_API const std::vector<std::shared_ptr<Function>>&
```

Comment:
Why did we change these to owning pointers?

Reply:
Because the FunctionType object captures them as such when an emitCall happens.